File: src/main/java/net/eduard/api/lib/menu/TradeType.java
package net.eduard.api.lib.menu;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public enum TradeType {
    SELABLE("Venda", "Vendivel"),
    BUYABLE("Compra", "Compravel"),
    BOTH("Troca", "Trocavel");
    private String displayName;
    private String description;
}

File: src/net/eduard/api/lib/menu/TradeType.kt
package net.eduard.api.lib.menu

enum class TradeType(var displayName: String,var description: String) {
    SELABLE("Venda", "Vendivel"), BUYABLE("Compra", "Compravel"), BOTH("Troca", "Trocavel");

    fun getName(): String {
        return displayName
    }
}